### PR TITLE
Sites Dashboard: Remove the max-width for the banner

### PR DESCRIPTION
**Pull request by okmttdhr**
Related to https://github.com/Automattic/dotcom-forge/issues/9915

## Proposed Changes

This PR is a follow-up for https://github.com/Automattic/wp-calypso/pull/96713#issuecomment-2497120519. 

Screenshot 2024-11-26 at 15 36 17

## Why are these changes being made?

pbxlJb-6DF-p2

## Testing Instructions

* Comment out [this line](https://github.com/Automattic/wp-calypso/blob/30d2f7cfb9ffc7b24edfa1d3d858ecc1bd2230d0/client/sites/components/sites-dashboard-banners-manager.tsx#L77) to show the banner on your local Calypso. (I wasn't sure how to show the banner.)
* Go to /sites
* Observe the banner is aligned with the "Sites" and DataView.
	* in the large view > 1440px
	* in the mobile view (especially > 430px)

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/sites/components/sites-dataviews/dataview-style.scss [hunk 1]
```diff
@@ -62,4 +62,29 @@
...
+.wpcom-site .main.a4a-layout.sites-dashboard {
+	.a4a-layout__top-wrapper,
+	.a4a-layout__body {
+		> * {
+			// To align with Core's hard-coded device width.
+			// https://github.com/WordPress/gutenberg/blob/ed66cc50e3c0b6785a48c15230c090790c0b0e6c/packages/dataviews/src/components/dataviews/style.scss#L84
+			// TODO: Remove when Core changes to use one of the predefined breakpoints.
+			@media (max-width: 430px) {
+				padding-inline: 24px;
+			}
```
Comment (okmttdhr) [inline client/sites/components/sites-dataviews/dataview-style.scss:76]: @oandregal 
This `430px` is derived from Core. Would it make sense to update it to `$break-mobile: 480px;`? If so, I’m happy to create a PR.

**Review by madhusudhand — APPROVED** [review]
Verified the banner in different viewports. It is set to the right width and matches sites header and dataview actions.

File: client/sites/components/sites-dataviews/dataview-style.scss (hunk 1 above)
Comment (oandregal) [reply client/sites/components/sites-dataviews/dataview-style.scss:76]: What I know is that it was introduced as part of [this PR](https://github.com/WordPress/gutenberg/pull/61333) (and then the code was moved around, so it was a bit difficult to track). I'll defer this one to designers. Can you raise this to them?

Comment (okmttdhr) [reply client/sites/components/sites-dataviews/dataview-style.scss:76]: Thanks for taking a look at it! Left a comment for now: https://github.com/WordPress/gutenberg/pull/61333#discussion_r1859672382

**Pull request merged**